cosmwasm/contracts/crosschain-registry/src/contract.rs

```diff
@@ -27,6 +30,15 @@ pub fn instantiate(
     let state = Config { owner };
     CONFIG.save(deps.storage, &state)?;
 
+    CHAIN_PFM_MAP.save(
+        deps.storage,
+        "osmosis",
```

Comment:
```suggestion
        // TODO: Move this to a constant to be source-chain
        "osmosis",
```

(Also doesn't this cause issues in the E2E tests?)

Reply:
I can add it to the config and set it to "osmosis" when the contract is initialized.

We want the proper name on the chain so that it can be referenced (for example when querying `getChannel("juno", "osmosis")`)

Reply:
added these as consts